hack/release.sh

```diff
@@ -26,8 +26,10 @@ declare -A COMPONENTS
 COMPONENTS=(
   ["channel-consolidated.yaml"]="config/channel/consolidated"
   ["channel-distributed.yaml"]="config/channel/distributed"
+  ["channel-crds.yaml"]="config/channel/resources"
```

Comment:
In which file the CRDs are currently at?

Reply:
it is defined in here: https://github.com/knative-sandbox/eventing-kafka/blob/main/config/channel/resources/kafkachannel-crd.yaml


And like on "eventing", we will have an extra file for _just_ the CRDs. See here as reference:
https://github.com/knative/eventing/blob/main/hack/generate-yamls.sh#L91-L94

This PR allows better reuse of just the CRDs